internal/data/mint_cota_kv_pair.go

```diff
@@ -74,32 +69,35 @@ func generateDefineWithdrawV1KvPairs(blockNumber uint64, entry biz.Entry, rp min
 			TokenIndex:           binary.BigEndian.Uint32(key.NftId().Index().RawData()),
 			OutPoint:             outpointStr,
 			OutPointCrc:          crc32.ChecksumIEEE([]byte(outpointStr)),
+			TxHash:               entry.TxHash.String()[2:],
 			State:                value.NftInfo().State().AsSlice()[0],
 			Configure:            value.NftInfo().Configure().AsSlice()[0],
 			Characteristic:       hex.EncodeToString(value.NftInfo().Characteristic().RawData()),
-			ReceiverLockScriptId: script.ID,
+			ReceiverLockScriptId: receiverLock.ID,
 			LockHash:             lockHashStr,
 			LockHashCrc:          lockHashCRC32,
+			LockScriptId:         senderLock.ID,
 			Version:              entry.Version,
 		})
 	}
 	return
 }
 
-func generateDefineWithdrawV0KvPairs(blockNumber uint64, entry biz.Entry, rp mintCotaKvPairRepo) (updatedDefineCotas []biz.DefineCotaNftKvPair, withdrawCotas []biz.WithdrawCotaNftKvPair, err error) {
+func generateMintV0KvPairs(blockNumber uint64, entry biz.Entry, rp mintCotaKvPairRepo) (defineCotas []biz.DefineCotaNftKvPair, withdrawCotas []biz.WithdrawCotaNftKvPair, err error) {
```

Comment:
Why change the name? this function is returning `DefineCotaNftKvPair ` and `WithdrawCotaNftKvPair `

Reply:
Mint is an action that will update `define` and `withdraw` SMT leaves. When we say minting CoTA NFTs,  the `DefineCotaNftKvPair` and `WithdrawCotaNftKvPair` must be updated. 

So I think `generateMintV0KvPairs ` is more simple and not less clear than `generateDefineWithdrawV0KvPairs`.

Reply:
ok, It's better to use the `action` to name other functions.

Reply:
Now the other functions are all already actions, such as `transfer/withdraw/define/claim`.